pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -325,7 +325,8 @@ func (rq *ResourceQuotaController) syncResourceQuota(v1ResourceQuota *v1.Resourc
 			Namespace:       resourceQuota.Namespace,
 			ResourceVersion: resourceQuota.ResourceVersion,
 			Labels:          resourceQuota.Labels,
-			Annotations:     resourceQuota.Annotations},
+			Annotations:     resourceQuota.Annotations,
```

Comment:
i am not aware of the plan to merge #45552, i am fine taking an alternate solution in the interim as this is a bug.